webtau-http/src/main/java/com/twosigma/webtau/http/datanode/StructuredDataNode.java

```diff
@@ -59,19 +59,66 @@ public DataNodeId id() {
     }
 
     @Override
-    public DataNode get(String name) {
+    public DataNode get(String nameOrPath) {
         if (isList()) {
-            return getAsCollectFromList(name);
+            return getAsCollectFromList(nameOrPath);
```

Comment:
now I got confused as to how this will work in case of `path`. `getAsCollectFromList` signature is also talking about `name` only